### ISSUE TITLE
intro text to be shown on page load (tbd via backend settings - projectInfo)

### ISSUE THREAD

**Issue by MauPalantir** [acdh-oeaw/vicav-vue3#197]

**Comment by dasch124** [comment]
@VeronikaEngler do we already have the text in TEI?

**Comment by VeronikaEngler** [comment]
yes, here: https://gitlab.oeaw.ac.at/acdh-ch/tunocent/tunocent-content/-/blob/master/vicav_texts/tunocent_opening_page.xml?ref_type=heads

**Comment by MauPalantir** [comment]
This is already supported by the system. I proposed a minor change in the config syntax [in vicav-app #113](https://github.com/acdh-oeaw/vicav-app/pull/113) but other than this decision, we are good.
